lib/Common/Exceptions/Throw.cpp

```diff
@@ -64,7 +64,7 @@ extern "C"{
 
 namespace Js {
 #if defined(GENERATE_DUMP) && defined(STACK_BACK_TRACE)
-    StackBackTrace * Throw::stackBackTrace = nullptr;
+    __declspec(thread) StackBackTrace * Throw::stackBackTrace = nullptr;
```

Comment:
Supported in clang under -fms-extensions mode.
http://clang.llvm.org/docs/AttributeReference.html#thread